openpype/hosts/houdini/api/lib.py

```diff
@@ -747,3 +753,48 @@ def get_camera_from_container(container):
 
     assert len(cameras) == 1, "Camera instance must have only one camera"
     return cameras[0]
+
+
+def update_job_var_context():
+    """Validate job path to ensure it matches the settings."""
+
+    project_settings = get_current_project_settings()
+    job_var_settings = \
+        project_settings["houdini"]["general"]["update_job_var_context"]
+
+    if job_var_settings["enabled"]:
+
+        # get and resolve job path template
+        job_path = StringTemplate.format_template(
+            job_var_settings["job_path"],
+            get_current_context_template_data()
+        )
+        job_path = job_path.replace("\\", "/")
+
+        if job_path == "":
+            # Set JOB path to HIP path if JOB path is enabled
+            # and has empty value.
+            job_path = os.environ["HIP"]
+
+        current_job = hou.hscript("echo -n `$JOB`")[0]
+
+        # sync both environment variables.
+        # because when opening new file $JOB is overridden with
+        # the value saved in the HIP file but os.environ["JOB"] is not!
+        os.environ["JOB"] = current_job
+
+        if current_job != job_path:
+            hou.hscript("set JOB=" + job_path)
+            os.environ["JOB"] = job_path
+
+            try:
+                os.makedirs(job_path)
+            except OSError as e:
+                if e.errno != errno.EEXIST:
+                    print(
+                        "  - Failed to create JOB dir. Maybe due to "
+                        "insufficient permissions."
+                    )
+
+            print("  - Context changed, update $JOB respectively to "
```

Comment:
```suggestion
            print("Updated $JOB to {}".format(job_path))
```
Anyone is free to call this function - thus context techhnically doesn't have to be changed for it to occur.

Reply:
i am actually wondering if it is possible to use logging here.

Reply:
> i am actually wondering if it is possible to use logging here. 

I tried using `log.debug` and `log.info` but `log` doesn't seem to log anything